src/components/checks/[share_token]/practice/NavigationButtons.tsx

```
@@ -0,0 +1,18 @@
+import { cn } from '@/src/lib/Shared/utils'
+
+export function PracticeNavigationNextButton({ ...props }: {} & ButtonHTMLAttributes<HTMLButtonElement>) {
+  const { nextQuestion } = usePracticeStore((store) => store)
+
+  return <Button {...props} onClick={() => nextQuestion(true)} children='Next' className={cn('', props.className)} />
+}
+
+export function PracticeNavigationPreviousButton({ ...props }: {} & ButtonHTMLAttributes<HTMLButtonElement>) {
+  const { previousQuestion } = usePracticeStore((store) => store)
+
+  return <Button {...props} variant='outline' onClick={() => previousQuestion(true)} children='Previous' className={cn('', props.className)} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)